google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -129,9 +133,13 @@ def pyarrow_timestamp():
         pyarrow.binary().id: "BYTES",
         pyarrow.string().id: "STRING",  # also alias for pyarrow.utf8()
         pyarrow.decimal128(38, scale=9).id: "NUMERIC",
+    }
+
+    if int(pyarrow.__version__.split(".")[0]) >= 3:
+        BQ_TO_ARROW_SCALARS["BIGNUMERIC"] = pyarrow_bignumeric
         # The exact decimal's scale and precision are not important, as only
         # the type ID matters, and it's the same for all decimal128 instances.
```

Comment:
Comment seems off, as this is a decimal256

Reply:
Copy-pasta, will fix it. Thanks for catching it!